### DISCUSSION TITLE
Understand Service/Error Code in Boiler

### DISCUSSION THREAD

**Discussion by bronkoo** [emsesp/EMS-ESP32#955] [Q&A] [answered]
Hi all,

my one and a half year old Buderus Logamax plus GB192i stopped working since 24h and the EMS-ESP Version v3.4.4

![Screenshot from 2023-02-13 00-35-16](https://user-images.githubusercontent.com/36919549/218344006-91a4f00d-24bb-47e1-9760-17b81b3868cf.png)

is showing in Boiler section **Service/Error Code 2945**:

![Screenshot from 2023-02-13 00-30-11](https://user-images.githubusercontent.com/36919549/218343966-269fb262-239a-4f58-841b-95dfcbd3e149.png)

I have no understand what is going on since I'm remote only currently.

Some HA screenshots additionally:

![Screenshot from 2023-02-13 00-41-14](https://user-images.githubusercontent.com/36919549/218344275-91492231-d77e-494b-8158-c11be76a5416.png)
![Screenshot from 2023-02-13 00-39-19](https://user-images.githubusercontent.com/36919549/218344277-97871348-9f68-451f-84fc-45e219354eb2.png)

Does anyone have some hints for a remote investigation on it or decoding the error?

**Comment by proddy** [comment, marked as answer]
does https://github.com/emsesp/EMS-ESP32/issues/938#issuecomment-1417950522 help?

**Reply by bronkoo** [reply]
Hi @proddy 

thanks a lot!
Based on your answer, decoding is no more a problem and additionally very nice thread to understand the integration in to HA.

But error code **2945** is missing, previously one **229** means bad flame... will try a power reset first, but need help on premises here...

**Reply by bronkoo** [reply]
Unlock on local display without power reset solved the issue, currently...

Service Codes flow: ->201 ->305 ->204 ->200
